lib/live-collection.js

```diff
@@ -255,6 +254,18 @@ module.exports = Backbone.Collection.extend({
     }, PATCH_TIMEOUT);
   },
 
+  resubscribeOnModelChange: function (model, prop){
+    var evtName = prop ? 'change:' + prop : 'change';
+    this.listenTo(model, evtName, this.resubscribe, this);
+  },
+
+  resubscribe: function (){
+    this.reset();
+    this.subscription.cancel();
+    this.subscription = null;
+    this.listen({ snapshot: true });
```

Comment:
Is it always the case when we resubscribe that we want a snapshot? Are some collections instantiated with `snapshot: false` for a specific reason? If so should we obey that here instead of just passing `true`?